packages/now-build-utils/src/detect-builders.ts

```diff
@@ -98,45 +95,63 @@ function detectFrontBuilder(
     config.framework = framework;
   }
 
-  if (devCommand) {
-    config.devCommand = devCommand;
+  if (projectSettings.devCommand) {
+    config.devCommand = projectSettings.devCommand;
+  }
+
+  if (projectSettings.buildCommand) {
+    config.buildCommand = projectSettings.buildCommand;
   }
 
-  if (buildCommand) {
-    config.buildCommand = buildCommand;
+  if (projectSettings.outputDirectory) {
+    config.outputDirectory = projectSettings.outputDirectory;
   }
 
-  if (outputDirectory) {
-    config.outputDirectory = outputDirectory;
+  if (pkg) {
+    const deps: PackageJson['dependencies'] = {
+      ...pkg.dependencies,
+      ...pkg.devDependencies,
+    };
+
+    if (deps['next']) {
```

Comment:
Why is `next` special cased here?

Reply:
@styfle because it uses the `@now/next` builder, for now